front/src/applications/stdcmV2/components/StdcmOrigin.tsx

```diff
@@ -46,6 +50,19 @@ const StdcmOrigin = ({ setCurrentSimulationInputs, disabled = false }: StdcmConf
     dispatch(updateOriginTime(e.target.value));
   };
 
+  const error = useMemo(() => {
+    if (!isNil(originDate) && !isNil(searchDatetimeWindow)) {
```

Comment:
Do you have a specific reason to use isNil here ? Or can we write it like this ?
```suggestion
    if (originDate && searchDatetimeWindow) {
```